models/experimental/llama2_70b/tests/unit_tests/test_reshape_rotary.py

```diff
@@ -94,10 +94,10 @@ def forward(self, xq, xk, rot_mat):
         )
 
         xq = tt_lib.tensor.transpose(xq, 1, 2)
-        xq = tt_lib.tensor.unpad(
+        xq = ttnn.slice(
             xq,
-            [0, 0, 0, 0],
-            [1 - 1, 8 - 1, 128 - 1, self.head_dim - 1],
+            ttnn.Shape([0, 0, 0, 0]),
+            ttnn.Shape([1 - 1, 8 - 1, 128 - 1, self.head_dim - 1]),
```

Comment:
Is this really a Shape? Or do we just use Shape as a list here?

Reply:
We should convert to a list, shape might have been okay before but now doesn't make sense since our shape objects include padding attribute

Reply:
@tt-aho good point, I just fixed it